app/serializable/serializable_submission.rb

```diff
@@ -13,4 +13,12 @@ class SerializableSubmission < JSONAPI::Serializable::Resource
   attribute :status do
     @object.aasm.current_state
   end
+
+  attribute :invoice_count do
+    @object.entries.invoices.count
+  end
+
+  attribute :order_count do
+    @object.entries.orders.count
+  end
```

Comment:
This overlaps a bit with the Submissions export. Does it make this [kind of query](https://github.com/dxw/DataSubmissionServiceAPI/blob/b1ecac8f1d1d5cc3cc7cfa0e57348b1fbdbf1094/app/models/export/submissions/extract.rb#L7-L15) unnecessary?

Reply:
(or are we planning a periodic batch job to update those fields in the API DB directly from `submission_entries`?)

Reply:
there's no plan to right now, but we could introduce a count cache on the submissions table at some point if we felt it made sense.